apps/hubble/src/hubble.ts

```
@@ -261,7 +270,12 @@ export class Hub implements HubInterface {
       lockMaxPending: options.commitLockMaxPending,
       lockTimeout: options.commitLockTimeout,
     });
-    this.engine = new Engine(this.rocksDB, options.network, eventHandler);
+    const mainnetClient = createPublicClient({
+      chain: mainnet,
+      transport: http(options.ethMainnetRpcUrl, { retryCount: 2 }),
+    });
```

Should hard fail if api key is not provided